### ISSUE TITLE
Working non electric bicycle failing to start

### ISSUE THREAD

**Issue by thedyze** [cataclysmbnteam/Cataclysm-BN#5505]
### Describe the bug

Sometimes when getting on a working non electric bicycle, it doesn't start.
Instead you get the 'car-fails-to-start' sound (which feels inappropriate) and you let go of the controls. After a few attempts it's usually possible to gain control. Unclear when this exactly happens. 
May have to do with pedals in less than stellar condition (see screenshot).

### Steps To Reproduce

1. Take control of a bicycle

### Screenshots

![image](https://github.com/user-attachments/assets/6d5660f7-2ac7-45bc-bf91-fa2c0d1800bf)
![image](https://github.com/user-attachments/assets/610639e5-bdee-4962-bf87-b2aa70db3dfc)

### Versions and configuration

Cataclysm BN version f393825 64-bit

Using the Otopack+ModsUpdates soundpack

**Comment by thedyze** [comment]
[save.zip](https://github.com/user-attachments/files/17264232/save.zip)

**Comment by ZChris13** [comment]
I don't think this is a bug, damaged foot pedals "failing to start" has always been in the game, so I can at least verify that this happens.
I guess your sound pack queuing the car-failed-to-start sound is a bug however.

**Comment by thedyze** [comment]
> I don't think this is a bug, damaged foot pedals "failing to start" has always been in the game, so I can at least verify that this happens. I guess your sound pack queuing the car-failed-to-start sound is a bug however.

Ahh, didn't even consider the soundpack being at fault. Yeah was discussing if it was a bug or feature on the discord.

**Comment by RoyalFox2140** [comment]
I don't know how bicycles work at all. Try it with an undamaged one.

**Comment by thedyze** [comment]
> I don't know how bicycles work at all. Try it with an undamaged one.

Yeah doesn't appear to happen with a mint condition bicycle. The car startup sound, is that something for the sound pack maintainer?

**Comment by RoyalFox2140** [comment]
Yeah that's for me then if you run Otopack or Fris if you can convince COAG pack to support BN. Use the otopack 2 issues or bug me on Discord.

**Comment by thedyze** [comment]
Will bug you

**Comment by RoyalFox2140** [comment]
It turns out this is a hardcoded issue and it sucks. Reopening because it requires code and I didn't think such a simple thing was hardcoded.

**Comment by ZChris13** [comment]
yeah, the sound pack stuff is a quite recent hack IIRC, there was no need to differentiate the different types of engines that could fail to start back in whalescata

**Comment by ghost** [comment]
I've done a lot more research here.

My preliminary guess is pedals_foot wasn't working and I needed to use variant: muscle instead. However, not only does the engine start work correctly on pedals, but I can't get a bicycle pedals to fail to start. I'll reopen if this ever gets reproduced again.

For reference I spawned a few mechanical bikes and smashed it with a hammer gradually and kept trying to farm out a fail state on a recent nightly. (Not quite latest, but nobody has touched this code in the past two or three weeks.

Also, yes. Engine failure is still hardcoded like this under clanking and stutter fail. I am of the opinion that bicycle pedals should never fail to start if you're still allowed to ride the bicycle. If such a case ever happens again either the code needs to not allow you to fail, or not allow you to attempt to ride saying it's too damaged. Closing as can't reproduce, but will reopen if we can reproduce.

```json
{
    "type": "sound_effect",
    "id": "vehicle",
    "variant": "engine_backfire",
    "volume": 70,
    "files": [ "vehicle/engine/engine_backfire.ogg" ]
  },
  {
    "type": "sound_effect",
    "id": "vehicle",
    "variant": "engine_clanking_fail",
    "volume": 50,
    "files": [ "vehicle/engine/engine_clanking_fail.ogg" ]
  },
  {
    "type": "sound_effect",
    "id": "vehicle",
    "variant": "engine_stutter_fail",
    "volume": 50,
    "files": [ "vehicle/engine/engine_stall_1.ogg" ]
  },
```
